config/v1/zz_generated.crd-manifests/0000_10_config-operator_01_apiservers-Default.crd.yaml

```diff
@@ -171,11 +172,6 @@ spec:
                         3. routes.route.openshift.io
                         4. oauthaccesstokens.oauth.openshift.io
                         5. oauthauthorizetokens.oauth.openshift.io
-                    enum:
-                    - ""
-                    - identity
-                    - aescbc
-                    - aesgcm
```

Comment:
Make sure this goes away when you fix the featureset to featuregate enum tag

Reply:
This is fixed now that I'm using FeatureGateAwareEnum.